src/tools/website/public/assets/translations/en.json

```diff
@@ -65,7 +65,7 @@
   "APP.HOME.LABEL.ORIENTATION.2.HEADLINE": "Use Elektra outside C/C++",
   "APP.HOME.LABEL.ORIENTATION.3.DETAILS": "Of course we can! An initiative like Elektra asks periodically for <a href=\"/decisions/readme\">decisions</a>. We wrote a <a href=\"https://book.libelektra.org\">book</a> explaining all the scientific background. We welcome everyone to join the <a href=\"https://github.com/ElektraInitiative/libelektra/issues\">discussions</a>!",
   "APP.HOME.LABEL.ORIENTATION.3.HEADLINE": "Can you Elaborate on the Why?",
-  "APP.HOME.LABEL.ORIENTATION.4.DETAILS": "Elektra is available in most distributions like <a href=\"https://admin.fedoraproject.org/pkgdb/package/elektra/\">Fedora</a>, <a href=\"http://packages.gentoo.org/package/app-admin/elektra\">Gentoo</a>, <a href=\"https://aur.archlinux.org/packages/elektra/\">Arch Linux</a>, <a href=\"https://packages.debian.org/de/jessie/libelektra4\">Debian</a>, <a href=\"https://launchpad.net/ubuntu/+source/elektra\">Ubuntu</a>, <a href=\"https://github.com/openwrt/packages/tree/master/libs/elektra\">Openwrt</a>, <a href=\"https://software.opensuse.org/package/elektra\">OpenSuse</a>, <a href=\"http://sophie.zarb.org/rpms/763d9e52beefaa15b1363d11d836b65c\">PLDLinux</a>, <a href=\"https://community.linuxmint.com/software/view/elektra-bin\">Linux Mint</a>, and <a href=\"https://openwrt.org/packages/pkgdata/libelektra-core?s[]=elektra\">OpenWRT</a>. See <a href=\"docgettingstarted/installation\">our installation instructions</a> for a full list.",
```

Comment:
As far as I understand it, the links should lead to a page about elektra for each of the distributions, not just to the links of the distributions themselves.
For Fedora and Gentoo, I couldn't find any, so as mentioned in #4099, the link might be removed.
For debian I found [a few](https://packages.debian.org/search?keywords=elektra&searchon=names&suite=all&section=all), I guess the binary one is the most suitable.
For sophie I also found [a few](http://sophie.zarb.org/search/results?search=elektra&x=0&y=0).
For both debian and sophie the used package is version 0.8.x. I'm unsure if suggesting to use these old versions is a good idea.

Reply:
Thank you! I will import these changes. 

My thought was to point at first to the Mainpage and if in the future the Package becomes available then change the Link to the right one.

Reply:
Yes, exactly. If there are no packages anymore, please remove the links. Otherwise link to the packages.